### PR TITLE
Farsi translation

### PR DESCRIPTION
**Pull request by Rahimz**
Farsi translation added to Django Girls Extensions tutorial.

### CODE REVIEW

**Comment by Rahimz** [comment]
Could someone help me why one of the check is failed?

File: fa/homework/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+با این روش، پست‌های جدید به جای آنکه بلافاصله منتشر شوند، به عنوان پیش‌نویس ذخیره می‌شوند و بعداً می‌توانیم آن‌ها را بازبینی کنیم. چیزی که الان لازم داریم راهی است که پست‌های پیش‌نویس شده را لیست کنیم و انتشار دهیم، برویم سراغش! 
+
+## صفحه‌ای با لیستی از پست‌های منتشر نشده 
+
+بخش مربوط به کوئری‌ست‌ها را به یاد می‌آورید؟ ما ویویی به نام `post_list` ساخته‌ایم که فقط پست‌های منتشر شده را نمایش می‌دهد (پست‌هایی که مقدار `published_date` برای آن‌ها خالی نیست).
+
+حالا زمان آن است که کار مشابهی را برای پست‌های پیش‌نویس انجام دهیم.
+
+حالا می‌خواهیم در هدر فایل `blog/templates/blog/base.html` یک لینک اضافه کنیم. از آنجا که  نمی‌خواهیم لیست پست‌های پیش‌نویس را به همه نشان بدهیم، پس این لینک رادر عبارت کنترلی {% if user.is_authenticated %} و دقیقاً بعد از دکمه مربوط به اضافه کردن پست جدید، اضافه می‌کنیم.
```
Comment (das-g) [inline fa/homework/README.md:21]: > Could someone help me why one of the check is failed?

The [build failure was](https://travis-ci.org/github/DjangoGirls/tutorial-extensions/builds/761286734#L394):
```
error: error while generating page "homework/README.md": 
Template render error: (/home/travis/build/DjangoGirls/tutorial-extensions/fa/homework/README.md)
  parseIf: expected elif, else, or endif, got end of file
The command "gitbook build" exited with 1.
```
The template tag syntax of HonKit overlaps with the template tag syntax of Django, so sometimes when we mention Django template tags, we have to put `{% raw %}…{% endraw %}` around them, to keep HonKit from trying to evaluate them. I guess that might be the case here, too, so try this:
```suggestion
حالا می‌خواهیم در هدر فایل `blog/templates/blog/base.html` یک لینک اضافه کنیم. از آنجا که  نمی‌خواهیم لیست پست‌های پیش‌نویس را به همه نشان بدهیم، پس این لینک رادر عبارت کنترلی {% raw %}`{% if user.is_authenticated %}`{% endraw %} و دقیقاً بعد از دکمه مربوط به اضافه کردن پست جدید، اضافه می‌کنیم.
```

Comment (Rahimz) [reply fa/homework/README.md]: @das-g  After I've made these changes, should I create a new pull request?

Comment (das-g) [reply fa/homework/README.md]: >  After I've made these changes, should I create a new pull request?

No, just tell us when this Pull Request here is ready for review.

**Comment by Rahimz** [comment]
@das-g I've added {% raw %} tag around any Django template tags that are placed  in the text.  It is ready to review.

File: en/heroku/README.md [hunk 2]
```diff
@@ -204,7 +204,7 @@ We can now visit the app in our browser with `heroku open`.
 
 This will open a url like [https://djangogirlsblog.herokuapp.com/]() in your browser, and at the moment you will probably see an error page.
 
-The error you saw was because we when we deployed to Heroku, we created a new database and it's empty. We need to run the `migrate` and `createsuperuser` commands, just like we did on PythonAnywhere.  This time, they come via a special command-line on our own computer, `heroku run`:
+The error you saw was because when we deployed to Heroku, we created a new database and it's empty. We need to run the `migrate` and `createsuperuser` commands, just like we did on PythonAnywhere.  This time, they come via a special command-line on our own computer, `heroku run`:
```
Comment (das-g) [inline en/heroku/README.md:207]: Good catch, but unrelated to the Farsi translation. Can you file this change as a separate Pull Request? Thanks!

File: fa/manual_pythonanywhere_deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
+# Deploying to PythonAnywhere manually
```
Comment (das-g) [inline fa/manual_pythonanywhere_deploy/README.md:1]: Is it intended that this file remained untranslated?

**Review by das-g — CHANGES_REQUESTED** [review]

File: en/heroku/README.md (hunk 2 above)
Comment (Rahimz) [reply en/heroku/README.md:207]: OK

File: fa/manual_pythonanywhere_deploy/README.md (hunk 3 above)
Comment (Rahimz) [reply fa/manual_pythonanywhere_deploy/README.md:1]: Ive tried to translate it but I saw some issues and see that in es translation it is not included so I left it untranslated.

**Comment by Rahimz** [comment]
@das-g I've completed my Farsi translation. Does it needed to make a new PR?

**Comment by das-g** [comment]
> @das-g I've completed my Farsi translation. Does it needed to make a new PR?

For the Farsi translation we can use this PR (#142), we don't need a separate one.

For the correction of the English original, please file a separate one. Please tell if you need help extracting that change to a separate branch.

**Comment by Rahimz** [comment]
I need help for extracting that change in English file, What is the best way to extract that? @das-g

**Comment by das-g** [comment]
> I need help for extracting that change in English file, What is the best way to extract that?

I don't know what's the _best_ way, but here's _a_ way that should do it:
:smiley:

### how to put the fix of the English original on a branch of its own

It seems c61b70d748a0e463fca1623ff5ac6518c28da74c is the only one of your commits that changes something in `en/`. Thus this should work:

In a newly created directory, outside of any existing repositories:
```bash
# Get a copy of the main repository for Django Girls tutorial extensions:
git clone git@github.com:DjangoGirls/tutorial-extensions.git
# or, if you haven't set up SSH:
git clone https://github.com/DjangoGirls/tutorial-extensions.git

# Change into the directory of that newly cloned repo:
cd tutorial-extensions

# Make your own "fork" on GitHub known to the local repo:
git remote add own git@github.com:Rahimz/tutorial-extensions.git
# or, if you haven't set up SSH:
git remote add own git@github.com:Rahimz/tutorial-extensions.git

# Make history and branches of own "fork" available locally:
git fetch own

# Create a new branch, based on "master" branch of main repo:
git switch origin/master -c fix-grammar --no-track
# (I've chosen "fix-grammar" as the branch name,
# as that seems to be what the change in question does.)

# Create a commit imitating c61b70d, but on that new branch.
git cherry-pick c61b70d748a0e463fca1623ff5ac6518c28da74c

# Optional: Change the commit message
# to better reflect the change made by that commit:
git commit --amend -m "remove gratuitous word"

# Push the result to your own "fork" on GitHub:
git push -u own fix-grammar
```
Part of the output is probably a URL to visit for creating a Pull Request from that branch.

### how to remove the change to the English original from this PR's branch

Only proceed if the above was successful. Once that's the case, we can continue in the same local repository.

The following assumes the same working directory as above after the command `cd tutorial-extensions`.
```bash
# Create a new (tracking) branch based on your "fork"'s master branch (= the branch of PR #142)
git switch own/master -c pr-142

# Create a new version of that branch's history, but without commit c61b70d:
git rebase --onto c61b70d748a0e463fca1623ff5ac6518c28da74c^ c61b70d748a0e463fca1623ff5ac6518c28da74c pr-142
## (the usual way would be with
# git rebase --interactive c61b70d748a0e463fca1623ff5ac6518c28da74c^
## but the variant above doesn't depend on what editor Git uses,
## so I thought it'd be more suitable for these instructions here.)

# Push the result to the master branch of your own "fork" on GitHub,
# thereby updating this Pull Request:
git push --force-with-lease
```

**Comment by Rahimz** [comment]
@das-g Thanks, but I encounter some issues so I reverted the change I've made in English file. I think you could merge this PR and then I will make a new PR for that typo.

**Comment by das-g** [comment]
Revert-Commits can be problematic for changes we _do_ want, just not in that part of the history. I've thus made the adaptions I've proposed and pushed them to your branch. (This was possible, because you've checked "Allow edits by maintainers" for this Pull Request.)

> I will make a new PR for that typo.

Please do. :+1:

If you want you can use commit 5fb2060ce397967487fe0739f0936ae782cf3d29 from [branch `fix-grammar` of my "fork"](https://github.com/das-g/tutorial-extensions/tree/fix-grammar) for that. If it's easier for you to make the change anew on a new branch, that works, too.

**Review by das-g — APPROVED** [review]

**Pull request merged**